packages/grafana-i18n/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "author": "Grafana Labs",
+  "license": "Apache-2.0",
+  "name": "@grafana/i18n",
+  "version": "12.1.0-pre",
+  "description": "Grafana Internationalization Library",
+  "keywords": [
+    "grafana",
+    "typescript"
+  ],
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/grafana/grafana.git",
+    "directory": "packages/grafana-i18n"
+  },
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "module": "src/index.ts",
+  "publishConfig": {
+    "main": "./dist/cjs/index.cjs",
+    "module": "./dist/esm/index.mjs",
+    "types": "./dist/cjs/index.d.cts",
+    "access": "public"
+  },
+  "files": [
+    "dist",
+    "./README.md",
+    "./CHANGELOG.md",
+    "LICENSE_APACHE2"
+  ],
+  "scripts": {
+    "build": "tsc -p ./tsconfig.build.json && rollup -c rollup.config.ts --configPlugin esbuild",
+    "bundle": "rollup -c rollup.config.ts --configPlugin esbuild",
+    "clean": "rimraf ./dist ./compiled ./package.tgz",
+    "typecheck": "tsc --emitDeclarationOnly false --noEmit",
+    "prepack": "cp package.json package.json.bak && node ../../scripts/prepare-npm-package.js",
+    "postpack": "mv package.json.bak package.json"
+  },
+  "dependencies": {
+    "i18next": "^24.0.0",
+    "i18next-browser-languagedetector": "^8.0.0",
+    "i18next-pseudo": "^2.2.1",
+    "react-i18next": "^15.0.0"
+  },
+  "devDependencies": {
+    "@grafana/tsconfig": "^2.0.0",
+    "@types/react": "18.3.18",
+    "rollup": "^4.22.4",
+    "typescript": "5.7.3"
+  },
+  "peerDependencies": {
+    "react": "^18"
```

Comment:
maybe ">=18" instead? or "18 || 19"? can't imagine we'll remember to update this when we move to react 19 😅

Reply:
Good catch 👍

Reply:
Fixed 👍